collect_app/src/main/java/org/odk/collect/android/widgets/interfaces/MediaWidgetDataRequester.java

```diff
@@ -0,0 +1,12 @@
+package org.odk.collect.android.widgets.interfaces;
+
+import android.content.Context;
+import android.net.Uri;
+
+import org.odk.collect.android.utilities.QuestionMediaManager;
+
+public interface MediaWidgetDataRequester {
+
+    String getUpdatedWidgetAnswer(Context context, QuestionMediaManager questionMediaManager, Object object,
```

Comment:
I'm not sure I understand what this interface is for? Could the implementation of `getUpdatedWidgetAnswer` just be a static helper that is shared between the widgets for the moment? I think it might make sense for there to be something that abstracts the `WaitingForDataRegistry` and the `QuestionMediaManager` but I'm not sure this feels right.

Reply:
Yes right it seems a bit confusing. More or less it calls other static members. Encapsulating all the calls in the method doesn't seem right to me. So I removed them.